recipes/rnasnp/meta.yaml

```diff
@@ -8,18 +8,19 @@ source:
   sha256: 4fb63b7ff149eca463e84e2f9e3c430e34d5e381dbda827b89e452408cead094
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - gcc   # [not osx]
     - llvm  # [osx]
+
   run:
     - libgcc    # [not osx]
 
 test:
-  commands:
-    - RNAsnp -h >/dev/null
+    commands:
+        - set +e && printf "A3G" > snp.txt && printf ">gi|56682960|ref|NM_000146.3| Homo sapiens ferritin, light polypeptide (FTL), mRNA\nGCAGTTCGGCGGTCCCGCGGGTCTGTCTCTTGCTTCAACAGTGTTTGGACGGAACAGATCCGGGGACTCTCTTCCAGCCTCCGACCGCCCTCCGATTTCCTCTCCGCTTGCAACCTCCGGGACCATCTTCTCGGCCATCTCCTGCTTCTGGGACCTGCCAGCACCGTTTTTGTGGTTAGCTCCTTCTTGCCAACCAACCA" > seq.fa && RNAsnp -f seq.fa -s snp.txt; echo "RNAsnp exite_code " $?
```

Comment:
Actually I have some doubts here. Imagine that the tool fails with a linkage error. This will yield some other exit code, but the test will still proceed.